sklearn/metrics/_ranking.py

```diff
@@ -636,6 +640,8 @@ def _multiclass_roc_auc_score(
 
     # validation for multiclass parameter specifications
     average_options = ("macro", "weighted", None)
+    if multi_class == "ovr":
+        average_options = ("micro",) + average_options
```

Comment:
Nit: to keep consistent ordering w.r.t the docstrings'.
```suggestion
    if multi_class == "ovr":
        average_options =  average_options + ("micro",)
```

Reply:
This change was introduced in [this commit](https://github.com/scikit-learn/scikit-learn/commit/ecf2b01de4bc9aaa1a1af119383834bf0cbff908) by @ogrisel to keep consistency with the order in the error message for other estimators, i.e. 'micro', 'macro', 'weighted', None.